examples/webgl_loader_vrml.html

```diff
@@ -1,129 +1,132 @@
-
-				controls.update();
-				renderer.render( scene, camera );
-
-				stats.update();
-
-			}
-
-		</script>
-
-	</body>
+<head>
+    <title>VRML loading with ThreeJs and VrmlParser</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <style>
+        body {
+            font-family: Monospace;
+            background-color: #000;
+            color: #fff;
+            margin: 0px;
+            overflow: hidden;
+        }
+
+        #info {
+            color: #fff;
+            position: absolute;
+            top: 10px;
+            width: 100%;
+            text-align: center;
+            z-index: 100;
+            display: block;
+        }
+
+        #info a, .button {
+            color: #f00;
+            font-weight: bold;
+            text-decoration: underline;
+            cursor: pointer
+        }
+
+        #viewpoints {
+            background-color: gray;
+            z-index: 101;
+            position: absolute;
+            width: 140px;
+            top: 10px;
+            left: 10px;
+            padding: 2px;
+            border: 1px solid white;
+        }
+
+        #viewpoints div {
+            width: inherit;
+        }
+    </style>
+
+</head>
+
+<body>
+<div id="info">
+    <a href="http://threejs.org" target="_blank">three.js / VrmlParser</a> -
+    vrml format loader test using VrmlParser -
+</div>
+<div id="viewpoints">
+    <h4>Viewpoints from the VRML file</h4>
+</div>
+
+<script src="../build/three.min.js"></script>
+<script src="js/libs/vrml.min.js"></script>
+
+<script src="js/loaders/VRMLLoader.js"></script>
+<script src="js/renderers/Projector.js"></script>
+<script src="js/controls/OrbitControls.js"></script>
+<script src="js/controls/FlyControls.js"></script>
+<script src="js/controls/FirstPersonControls.js"></script>
+<script src="js/controls/TrackballControls.js"></script>
+
+<script>
+
+	var camera, controls, scene, renderer, dirLight;
+
+	var animation;
+
+	// setup a clock
+	var clock = new THREE.Clock();
+
+	init();
+	animate();
+
+	function init() {
+		camera            = new THREE.PerspectiveCamera(60, window.innerWidth / window.innerHeight, 0.01, 1e5);
+		camera.position.z = 4;
+		camera.position.y = 3;
+		camera.position.x = 0;
+
+		scene = new THREE.Scene();
+
+		scene.fog = new THREE.Fog(0xffffff, 1, 5000);
+		scene.fog.color.setHSL(0.6, 0, 1);
+
+		scene.add(camera);
+
+		var debug = true;
+
+		// renderer
+		renderer                   = new THREE.WebGLRenderer({ antialias: true });
+		renderer.shadowMap.enabled = true;
+		renderer.setClearColor(0x000000, 1);
+		renderer.setSize(window.innerWidth, window.innerHeight);
+
+		// add support for animation and interaction
+		animation = new VrmlParser.Renderer.ThreeJs.Animation(debug);
+		animation.addClickSupport(camera, renderer);
+
+		var vrmlLoader = new THREE.VRMLLoader(new THREE.LoadingManager(), camera, controls, 'viewpoints', true);
```

Comment:
The controls, not 100% sure if still needed with the latest version.

camera and a viewpointsContainer id are needed to be able to present a list of ViewPoints that can be clicked to animate the camera to that ViewPoint and set properties such as field of view (currently not variable).